pkg/azureclients/armclient/azure_armclient.go

```diff
@@ -47,10 +65,43 @@ type Client struct {
 	regionalEndpoint string
 }
 
+func sender() autorest.Sender {
+	// note that we can't init defaultSenders in init() since it will
+	// execute before calling code has had a chance to enable tracing
+	defaultSenders.init.Do(func() {
+		// copied from http.DefaultTransport with a TLS minimum version.
+		transport := &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+			DialContext: (&net.Dialer{
+				Timeout:   30 * time.Second,
+				KeepAlive: 30 * time.Second,
+			}).DialContext,
+			ForceAttemptHTTP2:     true,
+			MaxIdleConns:          100,
+			MaxConnsPerHost:       100,
```

Comment:
what's the default values of these numbers, could you add a comment with a link?

Reply:
updated the comments.